### PR TITLE
JBIDE-27651: Update 4.19.0.AM1 Target Platform for 2021-03

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: util/verifyTarget.sh [hunk 1]
```diff
@@ -184,7 +183,8 @@ for PROJECT in $PROJECTS; do echo "Process $PROJECT ..."
   fi
 
   echo ""
-  if [[ ${includeSources} ]]; then
+  if [[ ! ${includeSources} ]]; then
```
Comment (jeffmaury) [inline util/verifyTarget.sh:186]: I don't get it

Comment (sbouchet) [reply util/verifyTarget.sh:186]: it's linked to the includeSources declaration above that i removed.
before, the includeSources was always present and the check was to inform if the mirror include or not the sources.
now, the logging is correct

Comment (sbouchet) [reply util/verifyTarget.sh:186]: if nothing is set, the log is including source and include sources is set
to exclude, the -x param is used and was incorrectly reporting including sources in log

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**